### ISSUE TITLE
`Find in Note` is the same as `Search Notes...`

### ISSUE THREAD

**Issue by dplanella** [Automattic/simplenote-electron#2762]
### Expected

When I go to `Edit` > `Find in Note`, I'd expect the search to be done only on the current note

### Observed

When I go to `File` > `Find in Note`, the search is done across all notes.

Effectively, there is no difference between the `Edit` > `Find in Note` and `Edit` > `Search Notes...` menu actions

### Reproduced

1. Start app
2. Go to `File` > `Find in Note` menu
3. Type a keyword to search on
4. Observe all matching keywords on the `All Notes` list being highlighted. The search is done across notes.

### Where did you see the bug

- System Make: N/A
- System Model: N/A
- OS: Ubuntu
- OS version: 20.04 
- Browser (if applicable): N/A (in app)
- Browser version (if applicable): N/A (in app)
- Simplenote app version: 2.7.1

**Comment by pachlava** [comment]
@dplanella thank you for the report! Since Simplenote currently provides only one kind of search, and it's global, I'd vote for removing the `Find in Note` menu item, and leave the `Search Notes...` only. 

Also, replacing the `Ctrl+Shift+S` shortcut for `Search Notes...` with `Ctrl+F` will be more consistent with macOS version shortcuts. But this will also affect the `Shortcuts` window.

@SylvesterWilmott would be great to hear your opinion!

**Comment by SylvesterWilmott** [comment]
>thank you for the report! Since Simplenote currently provides only one kind of search, and it's global, I'd vote for removing the Find in Note menu item, and leave the Search Notes... only.

I agree.

>Also, replacing the Ctrl+Shift+S shortcut for Search Notes... with Ctrl+F will be more consistent with macOS version shortcuts. But this will also affect the Shortcuts window.

On web we have intentionally avoided the "F" key but in the app we are free to use it for the search shortcut. Cmd + F currently focuses search for the mac installed Electron app and should be the same elsewhere.

**Comment by codebykat** [comment]
> On web we have intentionally avoided the "F" key

We're intentionally overriding the browser's search with `Ctrl+F` because the browser can't search the full text of a note (in other words, if we allow the default browser search to trigger, it will be broken -- it can only search within the visible portion of the note).

I think the `Ctrl+Shift+S` shortcut was left for historical purposes, or in case the OS/browser does take over `Ctrl+F`.

**Comment by sandymcfadden** [comment]
Do you think it is OK to remove the `CtrlOrCmd+Shift+S` from at least the electron app and update the Keyboard shortcut dialog to show `CtrlOrCmd+F` instead?

**Comment by codebykat** [comment]
Yeah, I would test it on the browser and see if we can remove it there as well. Let's continue listening for that shortcut in case Ctrl+F doesn't work in some cases, but if it tests okay, we can probably use that as the default.

**Comment by codebykat** [comment]
I just realized we have a decision in the other direction in #2599 (normalize on Ctrl+Shift+S). We should revisit that. I think this was for consistency across platforms?

**Comment by SylvesterWilmott** [comment]
According to the shortcuts spreadsheet, the change should be: Focus search = Cmd+Shift+F.

**Comment by erricgunawan** [comment]
A report from the forum:

https://forums.simplenote.com/forums/topic/find-in-note-strange-behaviour/?view=all#post-5623
